### PR TITLE
add crane container tools cli

### PR DESCRIPTION
**Pull request by DaveSys911**
Add crane cli utility package 
https://github.com/google/go-containerregistry/blob/main/cmd/crane/README.md

### CODE REVIEW

File: crane.hcl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+platform "Darwin" "x86_64" {
+  source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_${arch}.tar.gz"
+}
+
+platform "Darwin" "arm64" {
+  source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_${arch}.tar.gz"
+}
+
+platform "Linux" "x86_64" {
```
Comment (alecthomas) [inline crane.hcl:13]: Unfortunately this won't work as the CPU architecture key that Hermit supports is `amd64`, not `x86_64`.

You'll have to match on `amd64` here (and in the other x86_64 block), and replace `${arch}` in the URL with `x86_64`. eg.

```hcl
platform linux amd64 {
  source = source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_x86_64.tar.gz"
}
```

File: crane.hcl [hunk 2]
```diff
@@ -0,0 +1,26 @@
+description = "crane - Command line tool for interacting with remote images and registries."
+binaries = ["crane"]
+test = "crane version"
+
+platform "Darwin" "x86_64" {
+  source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_${arch}.tar.gz"
+}
+
+platform "Darwin" "arm64" {
```
Comment (alecthomas) [inline crane.hcl:9]: The OS keys here and elsewhere will all need to be lowercase.

**Review by alecthomas — CHANGES_REQUESTED** [review]
Thank you!

Needs a couple of tweaks to work, but otherwise LGTM!

File: crane.hcl (hunk 1 above)
Comment (DaveSys911) [reply crane.hcl]: Done

File: crane.hcl (hunk 2 above)
Comment (alecthomas) [reply crane.hcl]: eg. `"darwin"` and `"linux"`

**Comment by DaveSys911** [comment]
> Thank you!
> 
> Needs a couple of tweaks to work, but otherwise LGTM!

Thanks for the quick reply. Done- could u re-review and allow CI to run please?

**Pull request merged**